pkgs/applications/networking/irc/catgirl/default.nix

```
@@ -0,0 +1,21 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  name = "catgirl-1.6";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/catgirl/snapshot/${name}.tar.gz";
```

With the change from above

```suggestion
    url = "https://git.causal.agency/catgirl/snapshot/${pname}-$version{}.tar.gz";
```